modules/gov/msgs.go

```diff
@@ -124,6 +126,11 @@ func (msg MsgSubmitSoftwareUpgradeProposal) ValidateBasic() sdk.Error {
 	if err != nil {
 		return err
 	}
+	// if threshold not in [0.85,1), then print error
+	if msg.Threshold.LT(sdk.NewDecWithPrec(85,2)) || msg.Threshold.GTE(sdk.NewDec(1)){
```

Comment:
Why threshold must >= 85% but the flag threshold's default value is 0.667 ?

Reply:
Right, it doesn't make sense to have a default value of 0.667 while the valid range is [0.85,1) @MrXJC

Reply:
Update related docs as well: 0.667 --> 0.85 @MrXJC